### PR TITLE
simpcov_rm

### PR DESCRIPTION
**Pull request by Rileybmcc**
Make adjustments to allow better testing

### CODE REVIEW

File: spec/merchant_spec.rb [hunk 1]
```diff
@@ -15,7 +15,7 @@
 
   it 'returns id' do
     merchant = Merchant.new({
-      :id => 5,
+      :id => "5",
```
Comment (brennacodes) [inline spec/merchant_spec.rb]: this needs to remain an integer and not a string to match the interaction pattern below:
`m = Merchant.new({:id => 5, :name => "Turing School"})`

File: spec/customer_spec.rb [hunk 2]
```diff
@@ -5,7 +5,7 @@
 RSpec.describe Customer do
   let!(:time) {Time.now}
   let!(:customer) {Customer.new({
-    :id => 6,
+    :id => "6",
```
Comment (brennacodes) [inline spec/customer_spec.rb]: This needs to remain an integer to match the interaction pattern provided:
`c = Customer.new({
  :id => 6,
  :first_name => "Joan",
  :last_name => "Clarke",
  :created_at => Time.now,
  :updated_at => Time.now
})`

**Review by brennacodes — CHANGES_REQUESTED** [review]
Looks good! Thanks for your help! I left a couple of comments for tiny changes, but everything else looks great!

File: spec/merchant_spec.rb (hunk 1 above)
Comment (Rileybmcc) [reply spec/merchant_spec.rb]: Oops, must've forgotten to change that back

**Comment by brennacodes** [comment]
Looks good! Thanks @Rileybmcc!

**Comment by brennacodes** [comment]
OOps

**Pull request merged**